salt/cloud.sls

```diff
@@ -32,49 +32,41 @@ salt-cloud:
       {% endif %}
 {% endif %}
 
-{% for folder in salt_settings.cloud.folders %}
-{{ folder }}:
-  file.directory:
-    - name: /etc/salt/{{ folder }}
-    - user: root
-    - group: root
-    - file_mode: 744
-    - dir_mode: 755
-    - makedirs: True
-{% endfor %}
-
 {% for cert in pillar.get('salt_cloud_certs', {}) %}
 {% for type in ['pem'] %}
 cloud-cert-{{ cert }}-pem:
   file.managed:
-    - name: /etc/salt/cloud.providers.d/key/{{ cert }}.pem
+    - name: /etc/salt/pki/cloud/{{ cert }}.pem
```

Comment:
I mentioned it in the commit notes for f3ed6e, but it looks like I didn't specify why. Sorry.